user_templates/mcu.cfg

```diff
@@ -13,3 +13,28 @@
 
 # ------------------------------------------------------------------------------------------
 
+
+#-------------------------#
+#   TMCs DRIVERS          #
+#-------------------------#
+
+# ---------------------------------------------------------------------- AXIS DRIVERS ----> optional, but only one line need to be selected
+### 1. X Drivers -------------------------------------------------------------------------
+# [include config/hardware/axis/X/TMC/TMC2209.cfg]
+# [include config/hardware/axis/X/TMC/TMC2209_V0specific.cfg]
+### 2. Y Drivers -------------------------------------------------------------------------
+# [include config/hardware/axis/Y/TMC/TMC2209.cfg]
+# [include config/hardware/axis/Y/TMC/TMC2209_V0specific.cfg]
+### 3. Z Drivers -------------------------------------------------------------------------
+# [include config/hardware/axis/Z/TMC/TMC2209_1-Motor.cfg]
+# [include config/hardware/axis/Z/TMC/TMC2209_3-Motors.cfg]
+# [include config/hardware/axis/Z/TMC/TMC2209_4-Motors.cfg]
+# ----------------------------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------- EXTRUDER DRIVER ----> optional, but only one line need to be selected
```

Comment:
i think a little bit of more explanation regarding toolhead vs mcu e tmc should be added

Reply:
I don't think there is more explanation to add as when there is a toolhead MCU used and installed, it will inject some lines at the end that will automatically override the TMC pins. So this should be uncommented to select the extruder TMC and it doesn't matter if it's on the toolhead or the main MCU : it works for both the same way